tools/autograd/gen_python_functions.py

```diff
@@ -61,6 +61,7 @@
     '_cumsum.*', '_cumprod.*', '_sum.*', '_prod.*',
     '_th_.*', '_thnn_.*',
     'arange.*', 'range.*', '_solve.*', '_inverse.*',
+    'full.names', 'full.out', 'full',
```

Comment:
just pattern matching here, I think this should be:
`full_out`, `full` -- I don't think you need names and I think the `.` is just some regex thing that ends up the same -- it's not actually triggering off the qualified name you are giving, I think.

Reply:
You are (as usual) correct. My mistake. Fixed to "full(_out)?", which is consistent with other registrations (for example randint(_out)? below).